api/demo/routes.py

```diff
@@ -61,20 +63,20 @@ def authorized():
 def logout():
     session.clear()  # Wipe out user and its token cache from session
     return redirect(  # Also logout from your tenant's web session
-        config.AUTHORITY
+        env.config.get("AZURE_AD_AUTHORITY")
```

Comment:
💯 Nice config use - lets discuss how you found this method :)

Reply:
I wanna know it too 😄

Reply:
Great yes more to discuss